rush.json

```diff
@@ -40,8 +40,13 @@
      *
      * The default value is false to avoid legacy compatibility issues.
      * It is strongly recommended to set strictPeerDependencies=true.
+     *
+     * NOTE: Changed to false due to sudden error:
+     *
+     *     ERROR  @rush-temp/abort-controller > eslint > espree: acorn-jsx@5.2.0 requires a peer of acorn@^6.0.0 || ^7.0.0 but none was installed.
```

Comment:
@mikeharder is aware of this and of the api-extractor issue we're having. The underlying problem is unrelated to this hotfix.

Reply:
Seems like `rush install` cannot be trusted to work over time. This is a separate problem that Mike is investigating, unrelated to this hotfix.